### PR TITLE
V2.0 teaser

### PR DESCRIPTION
**Pull request by aliashori75**

### CODE REVIEW

**Comment by aliashori75** [comment]
> _ No description provided. _

**Comment by aliashori75** [comment]
> > _ No description provided. _

**Comment by aliashori75** [comment]
img_raw = tf.io.read_file(img_path)
img = tf.io.decode_image(img_raw)
img = tf.image.convert_image_dtype(img, tf.float32)
img = tf.image.resize(img, [500,500])

plt.title("TensorFlow Logo with shape {}".format(img.shape))
_ = plt.imshow(img)

**Comment by aliashori75** [comment]
> img_raw = tf.io.read_file(img_path) img = tf.io.decode_image(img_raw) img = tf.image.convert_image_dtype(img, tf.float32) img = tf.image.resize(img, [500,500])
> 
> plt.title("TensorFlow Logo with shape {}".format(img.shape)) _ = plt.imshow(img)

@aliashori75

**Comment by aliashori75** [comment]
> _ No description provided. _

import tensorflow as tf
mnist = tf.keras.datasets.mnist

(x_train, y_train),(x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='relu'),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

model.fit(x_train, y_train, epochs=5)
model.evaluate(x_test, y_test)

**Comment by aliashori75** [comment]
> _ No description provided. _

@aliashori75

**Review by aliashori75 — COMMENTED** [review]
@aliashori75
